model-optimizer/extensions/front/tf/sparse_weighted_sum.py

```diff
@@ -28,6 +28,7 @@ class ExperimentalSparseWeightedSumFrontReplacer(FrontReplacementSubgraph):
     Such sub-graph is met in the Wide and Deep model in case of the SINGLE categorical feature.
     """
     enabled = True
+    force_clean_up = True
```

Comment:
It is not allowed to run cleanup on the front phase, because this change breaks conversion of TensorFlow Object detection API models.

You already remove some nodes using the `graph.remove_nodes_from` function. Just add matched 'Where' node to this list.

Reply:
done